app/assets/images/staggered-dev.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="106" height="103" viewBox="0 0 106 103" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <g clip-path="url(#clip0)">
+    <rect x="64.3086" y="8" width="48" height="48" rx="3" transform="rotate(30.4274 64.3086 8)" fill="#1AB3A6" />
+    <path d="M65.9833 40.8852C65.3197 42.0149 64.4089 42.7303 63.251 43.0314C62.1032 43.3298 60.9387 43.132 59.7572 42.4381L55.9612 40.2086L63.9239 26.6514L67.7974 28.9264C68.9272 29.59 69.6393 30.4988 69.9339 31.653C70.2388 32.8045 70.0594 33.9452 69.3958 35.0749L65.9833 40.8852ZM63.5623 39.4633L66.9749 33.653C67.2593 33.1689 67.3102 32.7125 67.1277 32.2841C66.949 31.8492 66.6983 31.5369 66.3755 31.3473L64.9229 30.4942L59.8041 39.2095L61.2566 40.0627C61.5794 40.2523 61.9724 40.3224 62.4354 40.2731C62.9023 40.2174 63.2779 39.9475 63.5623 39.4633ZM71.5427 49.3602L66.4588 46.3742C65.9746 46.0898 65.6659 45.6914 65.5327 45.179C65.3995 44.6667 65.4751 44.1684 65.7595 43.6842L71.7315 33.5163C72.0158 33.0321 72.4142 32.7234 72.9266 32.5902C73.439 32.457 73.9373 32.5326 74.4214 32.817L79.5054 35.803L78.0835 38.2239L73.7258 35.6645L71.8773 38.8117L74.5404 40.3758L73.1185 42.7967L70.4554 41.2326L68.607 44.3798L72.9646 46.9392L71.5427 49.3602ZM77.6826 52.9664C76.8756 52.4924 76.5618 51.5483 76.7412 50.1342L80.5613 36.4232L83.2243 37.9873L80.1738 48.7249L88.0662 40.8311L90.7292 42.3952L80.6147 52.4092C79.4669 53.2546 78.4896 53.4403 77.6826 52.9664Z" fill="white" />
+    <rect x="2.49121" y="59.0127" width="48" height="48" rx="3" transform="rotate(-32.6668 2.49121 59.0127)" fill="#08090A" />
+    <path d="M32.5745 72.4007C33.2817 73.5037 33.5075 74.6396 33.2519 75.8084C32.9986 76.9669 32.2953 77.916 31.1419 78.6555L27.4359 81.0317L18.9496 67.7959L22.7312 65.3713C23.8342 64.6641 24.9669 64.4403 26.1295 64.6999C27.2943 64.9491 28.2303 65.6253 28.9375 66.7282L32.5745 72.4007ZM30.211 73.9161L26.574 68.2437C26.2709 67.771 25.887 67.519 25.4224 67.4879C24.9537 67.4504 24.5617 67.5327 24.2466 67.7348L22.8285 68.644L28.284 77.1527L29.7021 76.2435C30.0172 76.0414 30.2576 75.7228 30.4232 75.2875C30.5848 74.8459 30.5141 74.3888 30.211 73.9161ZM42.6479 71.2782L37.6845 74.4606C37.2118 74.7636 36.7168 74.8586 36.1997 74.7456C35.6825 74.6325 35.2723 74.3396 34.9692 73.8669L28.6045 63.9401C28.3014 63.4674 28.2064 62.9725 28.3195 62.4553C28.4326 61.9381 28.7255 61.5279 29.1982 61.2249L34.1615 58.0425L35.677 60.406L31.4226 63.1337L33.3927 66.2063L35.9925 64.5394L37.508 66.9029L34.9081 68.5698L36.8781 71.6424L41.1325 68.9147L42.6479 71.2782ZM48.6421 67.4348C47.8543 67.94 46.8704 67.7926 45.6905 66.9927L35.1924 57.3815L37.7923 55.7146L45.9871 63.294L42.5193 52.6837L45.1192 51.0168L49.4722 64.568C49.7067 65.9741 49.43 66.9297 48.6421 67.4348Z" fill="white" />
+  </g>
+  <defs>
+    <clipPath id="clip0">
+      <rect width="106" height="103" fill="white" />
+    </clipPath>
+  </defs>
+</svg>
```

Comment:
we should optimize this svg (with `svgo`?) if it hasn't been optimized already

Reply:
TIL! updated! 21% reduction

Reply:
oh yeah, it's amazing! gotta save space where you can 😅